tools/topology/topology2/include/pipelines/cavs/dai-copier-eqiir-gain-module-copier-capture.conf

```diff
@@ -0,0 +1,204 @@
+#
```

Comment:
@singalsu if this PR is not too urgent, can it wait till the new pipeline extension template goes upstream?